addons/stock/models/stock_move.py

```diff
@@ -815,12 +824,22 @@ def _update_reserved_quantity(self, need, available_quantity, location_id, lot_i
 
         taken_quantity = min(available_quantity, need)
 
+        # `taken_quantity` is in the quants unit of measure. There's a possibility that the move's
+        # unit of measure won't be respected if we blindly reserve this quantity, a common usecase
+        # is if the move's unit of measure do not allow fractionnal reservation. We chose to
```

Comment:
does not / fractional  / maybe talk about roundings not respecting....    (if the roudings of the uoms don't allow...)